Dockerfile

```diff
@@ -343,13 +358,24 @@ RUN mkdir -p ${GENIE} &&\
       --disable-lhapdf5 \
       --enable-gfortran \
       --with-gfortran-lib=/usr/x86_64-linux-gnu/ \
-      --disable-pythia8 \
-      --with-pythia6-lib=${__prefix}/pythia6 \
+      --enable-pythia8 \
+      --with-pythia8-lib=${__prefix}/lib \
       --enable-test \
     && \
     make -j$NPROC && \
     make -j$NPROC install
 
+#Unfortunately ... need to use the master branch of GENIE reweight...
```

Comment:
As discussed in the SW Dev meeting, we should change this to be a specific commit sha and if possible a specific human-readable tag made by the GENIE folks. Then (whichever path occurs), we should update `ENV` and `LABEL` with the version of GENIE Reweight installed.

something like

```
ENV GENIE_REWEIGHT_VERSION=1_02_03
LABEL genie_reweight.version=${GENIE_REWEIGHT_VERSION}
```